media-gfx/xpaint/files/xpaint-3.1.3-cflags-ldflags-strip.patch

```diff
@@ -0,0 +1,38 @@
+--- a/vxp2ps/Makefile	2021-01-19 10:35:54.000000000 +0200
++++ b/vxp2ps/Makefile	2021-03-21 10:16:05.028048054 +0200
+@@ -3,16 +3,16 @@
+ all: vxp2ps vxp2tex vxp2dkw
+ 
+ vxp2ps: vxp2ps.c
+-	gcc vxp2ps.c -o vxp2ps -lm $(CFLAGSEXTRA)
++	gcc vxp2ps.c -o vxp2ps -lm $(CFLAGS) $(CFLAGSEXTRA) $(LDFLAGS)
```

Comment:
CPPFLAGS after $(CC) and possibly kill CFLAGSEXTRA (what is it?)

Reply:
`CFLAGSEXTRA` is a dependency to `libdvipgm` we don't have for now, so it expands to nothing.  It can be added later if needed.